.github/workflows/docker-iam-rucio-sync.yml

```diff
@@ -13,10 +8,11 @@ on:
       - containers/iam-rucio-sync/**
```

Comment:
I suggest we build on push to the `main` branch and add a condition for the tag (see below).